app/views/products/show.html.erb

```diff
@@ -56,6 +56,15 @@
                 <th>Category</th>
                 <td><%= @product.category&.name %></td>
               </tr>
+              <tr>
+                 <th>QR Code</th>
+                <td style='text-align: center'>
+                <%= image_tag(@png.to_data_url, size: '120x120') %><br>
```

Comment:
Can we call it `@qrcode_png` instead? Just @png is not a good variable name because it tells you what the format is but it doesn't tell you _what_ it is